### DISCUSSION TITLE
Demo

### DISCUSSION THREAD

**Discussion by jlharper59** [openaudible/openaudible#714] [General]
Just ensuring that the download without a license lets me basically download books and view the functionality of the program, but NO conversion at all?

**Comment by openaudible** [comment]
Correct. You can download your books, and convert them once you have a license.  A few other features are locked as well.

**Reply by izzy** [reply]
That is kinda bs and should be the first thing your website says. When I read "open" as part of an app's name I don't expect the first thing when trying out the software to be a "You need a paid license for that". It's plan misleading and prompted me to uninstall the whole thing out of spite. I mean, imagine having your username being part of such a bad UX that people come here just to comment this..

**Comment by halloleo** [comment]
Agreeing with @izzy . It is terrible to have a commercial product riding the Open Source wave. Please get a new domain on .com and state clearly and first thing on the website that this is no open source product any more. Feels like you are heating.

I totally understand your decision to close the source. But *please* be a lot more transparent about. Else you won't get many users form the GitHub community.

**Comment by openaudible** [comment]
Open Source Wave? When we were open source we got no source code contributions and just a few donations. 
The .com domain is not available and don't know how to contact the person who owns it. That would be nice to have.
As far as the old source code goes, we could delete it, move it to another repo, or do nothing. 
We use github to track and address issues. Almost 900 issues to date and dozens of releases. 
It sounds like you're just sad you can't have something for free. Of course, you're free to fork one of the other free software alternatives or start where the old code left off. That's the joy of Open Source. Let us know how it goes.

**Reply by halloleo** [reply]
> Open Source Wave? When we were open source we got no source code contributions and just a few donations.

I mean with "open source wave" that you benefit from the positive reputation open source has. People see: This is on GitHub, it has recent GitHub issues, so it is _community_ maintained, so it is worth trying out. (I don't mean that you necessarily get lots of people contributing.)

> It sounds like you're just sad you can't have something for free

Non, no, that's not it. I just criticise that -- without looking very closely -- the project _appears_ as something (an open source project) which it is not. -- You should make your move to closed source, commercial software ***much*** clearer!
